daml-lf/data/src/test/scala/com/digitalasset/daml/lf/data/RefTest.scala

```diff
@@ -399,20 +474,20 @@ class RefTest extends AnyFreeSpec with Matchers with TableDrivenPropertyChecks w
         } ((x compare y) == 0) shouldBe (x == y)
       }
 
-      "is reflexive" - {
+      "is reflexive" in {
```

Comment:
@remyhaemmerle-da These tests were not running. It's a pity that the `AnyFreeSpec` allows this. Enabled now

Reply:
:bow: